basic-examples/bind-constants/subworkflow.yaml

```diff
@@ -1,3 +1,5 @@
+# This workflow accepts fields for an integer and 
+# an object (containing your repeated values).

```

Comment:
This text describes the inputs, but it doesn't describe the output, and it doesn't mention what the purpose or effect of this workflow is.  (And, in the context of this workflow, the concept of "repeated" is kind of unclear.)

I would be inclined to pillage your text from the `README.md`:
> This workflow uses a plugin which returns the value of the `name` input as its output.  The other input values are bundled with the plugin output as this subworkflow's output. This workflow serves as a stand-in for what in a real world scenario would be a more complex operation.

Reply:
copied